apps/voting-disputable/app/src/components/VoteCard/VoteCard.js

```diff
@@ -41,8 +41,7 @@ function VoteCard({ vote, onOpen }) {
   } = vote
 
   //TODO: Remove this once we have real data
-  vote.disputable = getDisputableVoteById(voteId)
-
+  vote.disputable.action = getDisputableVoteById(voteId)
```

Comment:
Tiny nitpick: should we rename this temp func to be `getMockVoteActionById` or similar? and update the mock references to be `voteAction` rather than `vote`. Really not a big deal! just took me a second to see which parts were being mocked vs real.